storybookjs/storybook
# CLI: Don't output startup information on smoke test

Issue: https://github.com/storybookjs/storybook/issues/5164

Don't output the startup information when running a smoke test.

## What I did
Moved the function call into the condition so it's not showing when running a smoke test.

The snapshots were failing prior to my changes, so I updated them in order to run the tests successfully.

## How to test

- Run the CLI command without the smoke test flag, startup information should be available.
- Run the CLI command with the smoke test flag, you should see only the stats, but not the startup information.
